app/build.gradle

```diff
@@ -152,16 +152,15 @@ dependencies {
 
     // Testing dependencies
```

Comment:
Just to be organized, I believe we should put all of the `Testing Dependencies` at the bottom. What do you think?

Reply:
👍

Reply:
I like it!